### PR TITLE
Add updateBreadcrumbs hook to GridFieldDetailForm

### PR DESCRIPTION
**Pull request by wilr**
Requires: https://github.com/silverstripe/silverstripe-admin/pull/291

### CODE REVIEW

File: src/Forms/GridField/GridFieldDetailForm_ItemRequest.php [hunk 1]
```diff
@@ -587,10 +587,11 @@ public function Breadcrumbs($unlinked = false)
 
         /** @var ArrayList $items */
         $items = $this->popupController->Breadcrumbs($unlinked);
+
         if ($this->record && $this->record->ID) {
             $title = ($this->record->Title) ? $this->record->Title : "#{$this->record->ID}";
             $items->push(new ArrayData(array(
-                'Title' => $title,
+                'Title' => DBField::create_field('Varchar', $title),
```
Comment (tractorcow) [inline src/Forms/GridField/GridFieldDetailForm_ItemRequest.php]: We don't need to explicitly cast anything that's the same as the default_cast.

**Comment by tractorcow** [comment]
There are some PHPCS issues. I'm looking at behat.

**Comment by wilr** [comment]
Linting issues? It's 1 line?

**Comment by tractorcow** [comment]
That'll make it easy to fix. :)

**Pull request merged**